scripts/version-urls

```diff
@@ -19,6 +19,6 @@ process.stdout.write(fs.readFileSync(process.argv[2], 'utf8').replace(
              '/tree/v' + deps[name] + parts.slice(1).join('');
     }
     process.stderr.write('Exact version not specified for ' + name + '\n');
-    process.exit(1);
+//  process.exit(1);
```

Comment:
Yes, for the time being. We can't yet specify versions for `sanctuary-{either,maybe}` as these packages have not yet been published.